libs/utils/utils.js

```diff
@@ -1153,3 +1153,5 @@ export function loadLana(options = {}) {
   window.addEventListener('error', lanaError);
   window.addEventListener('unhandledrejection', lanaError);
 }
+
+export const reloadPage = () => window.location.reload();
```

Comment:
since this is used only in upgrade, flow I think should be kept in-place.

Reply:
its to be used also in other files. placing it in utils allows us to stub it.